isos/bootstrap/bootstrap

```diff
@@ -43,7 +38,7 @@ if mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}; then
     # enable full system functionality in the container
     echo "Publishing modules within container"
     mkdir -p ${MOUNTPOINT}/lib/modules
-    mount --bind /lib/modules ${MOUNTPOINT}/lib/modules
+    cp -pr /lib/modules/* ${MOUNTPOINT}/lib/modules/
```

Comment:
We do if we want to provide more (like nfs/cifs mounting, enabling some other kernel feature etc)